eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please add linter config for html+css+js from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link. Currently, you have not configured linter for JavaScript.

Reply:
I dont understand. I followed all the steps provided in the link